utils/config/daos_server.yml

```diff
@@ -68,7 +68,6 @@
 #fault_cb: ./.daos/fd_callback
```

Comment:
While you are there, could you please change number of targets in this file to 16 instead of 20?